### ISSUE TITLE
panel.qte fails when data is a tbl_df

### ISSUE THREAD

**Issue by lrose1** [bcallaway11/qte#3]
Perhaps this is not an issue, but this took me a long time to figure out.

```
#load the package
library(qte)

#load the data
data(lalonde)
lalonde_tbl <- as_tibble(lalonde.psid.panel)

#Run the panel.qtet method on the observational data with no covariates
pq1 <- panel.qtet(re ~ treat, t=1978, tmin1=1975, tmin2=1974, tname="year",
                  x=NULL, data=lalonde_tbl, idname="id", se=FALSE,
                  probs=seq(0.1, 0.9, 0.1))
```

`Error in `$<-.data.frame`(`*tmp*`, "w", value = 1) : 
  replacement has 1 row, data has 0
In addition: Warning message:
In panelize.data(data, idname, tname, t, tmin1, tmin2) :
  dropping 8025 observations that are not in period: 1978, 1975, 1974...`

**Comment by mel0105** [comment]
Hi. I have the same problem. Did you find a solution?

**Comment by lrose1** [comment]
@mel0105 Yes, just change it to a data.frame. Bit of a trivial issue, but the error message wasn't informative so it took me a while.

**Comment by mel0105** [comment]
oh ok! I'll try.

**Comment by mel0105** [comment]
<img width="617" alt="Screenshot 2024-02-15 at 09 29 23" src="https://github.com/bcallaway11/qte/assets/70906131/e5bc7c31-e224-4152-9ace-c6c730efaf19">

do you know how to solve this?
